api/src/OpenTelemetry/Internal/Common/Types.hs

```diff
@@ -10,7 +10,13 @@ module OpenTelemetry.Internal.Common.Types (
   AnyValue (..),
   ToValue (..),
   ShutdownResult (..),
+  takeWorseShutdownResult,
+  takeWorstShutdownResult,
+  flushResultToShutdownResult,
   FlushResult (..),
```

Comment:
This is a bit of a driveby request on my part, but it would be nice to reexport `FlushResult` so that callers of [forceFlushTracerProvider](https://hackage.haskell.org/package/hs-opentelemetry-api-0.1.0.0/docs/OpenTelemetry-Trace-Core.html#v:forceFlushTracerProvider) can actually use the type.